tools/authConfig.js

```diff
@@ -1,3 +1,5 @@
+// Recommend getting rid of this... get db config from environment variables
+// get secret from another environment variable - Harold
```

Comment:
Not sure what environment variables are. this code is from the authentication tutorial- pure voodoo